frontend/src/Explorer/Util/String.js

```diff
@@ -9,3 +9,9 @@ exports.substituteImpl = function(str) {
     });
   };
 };
+
+exports.take = function (n) {
```

Comment:
Because we then need to install purescript-strings and when you try to do that you have version incompatibility (you have to choose packages).